repotracker/repotracker.go

```diff
@@ -828,55 +826,68 @@ func createVersionItems(ctx context.Context, v *model.Version, ref *model.Projec
 		}
 		_, err = evergreen.GetEnvironment().DB().Collection(model.VersionCollection).InsertOne(sessCtx, v)
 		if err != nil {
-			_ = sessCtx.AbortTransaction(sessCtx)
-			grip.Notice(message.Fields{
+			grip.Error(message.Fields{
```

Comment:
I think notice is still the right level to log at, since there are sporadic things that will cause us to retry